### DISCUSSION TITLE
2 wallets / fingerprints after a fresh installation

### DISCUSSION THREAD

**Discussion by JoJ123** [Chia-Network/chia-blockchain#5388] [Support]
Hi,
all started with the Windows GUI.
1. Plotting & Farmin (Windows GUI)
2. Farming in a docker container
   a. I copied all the mainnet folder and was directly synced
   b. I won my first 2 XCH on this setup.
   c. After update to 1.1.5 I was always getting out of sync.
3. I decided to setup a VM with a fresh start.
   a. Setup on Ubuntu with version 1.1.5 and the 24 words and a fresh "chia init"
   b. After 2 days I'm now again fully synced and also farming now.
   c. When I type "chia wallet show" I have to pick a "wallet key"
   d. When I type "chia keys show" I see two fingerprints, one is the same that is displayed on the Chia GUI in Windows, the other one is new
  
My Questions:
Why I have two wallets / keys set? Should I get rid of one?
Why the wallets both show "height: 0" and don't sync :(

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
